packages/fast-components-react-msft/app/app.tsx

```diff
@@ -25,10 +29,12 @@ export default class App extends React.Component<{}, {}> {
     public render(): JSX.Element {
         return (
             <Site
-                title={"FAST Microsoft component documentation"}
                 formChildOptions={formChildOptions}
                 onUpdateDirection={this.handleUpdateDirection}
             >
+                <SiteTitle slot={"title"}>
+                    <SiteTitleBrand>FAST</SiteTitleBrand> Microsoft component documentation
```

Comment:
We may want to check, I think one of the things Jason wanted was to shorten this string a bit - perhaps even to just "FAST documentation" for now (since it's currently the only one up)

Reply:
+1 for shorter

Reply:
resolved